ooler_mqtt_bridge

```diff
@@ -87,8 +87,11 @@ async def control_temperature(mqtt: Client, myooler: ooler.Ooler) -> None:
         async for message in messages:
             if message.topic.matches(message_filter):
                 temperature = int(float(message.payload.decode()))
-                await myooler.set_desired_temperature_c(temperature)
-                await send_update(mqtt, myooler)
+                temp_unit = await myooler.get_temperature_unit()
+                if temp_unit is constants.TemperatureUnit.Fahrenheit:
+                    await myooler.set_desired_temperature_f(temperature)
+                else:
+                    await myooler.set_desired_temperature_c(temperature)
```

Comment:
Was getting rid of the `send_update` here intentional?

Reply:
No :facepalm:

Reply:
```suggestion
                    await myooler.set_desired_temperature_c(temperature)
                await send_update(mqtt, myooler)
```